app/gui/src/controller/searcher.rs

```diff
@@ -1188,6 +1256,11 @@ fn component_list_builder_with_favorites<'a>(
         builder = builder.with_local_scope_module_id(id);
     }
     builder.set_grouping_and_order_of_favorites(suggestion_db, groups);
+    let base_lib_qn = project::QualifiedName::of_standard_base_library();
+    let input_group_name = INPUT_COMPONENT_GROUP_NAME;
+    let input_group_qn = component::group::QualifiedName::new(base_lib_qn, input_group_name);
+    let snippets = LITERAL_INPUT_NODES_SNIPPETS.with(|c| c.clone());
```

Comment:
what is `c`? We are sometimes using one-letter variables if it is easy to understand what it is from the context. Here, I can't figure what `c` means.